src/context/directory/handlers/prompts.js

```diff
@@ -0,0 +1,37 @@
+import path from 'path';
+import {
+  existsMustBeDir, isFile, dumpJSON, loadJSON
+} from '../../../utils';
+
+function parse(context) {
+  const baseFolder = path.join(context.filePath);
+  if (!existsMustBeDir(baseFolder)) return {}; // Skip
+
+  const promptsFile = path.join(baseFolder, 'prompts.json');
+
+  if (isFile(promptsFile)) {
+    /* eslint-disable camelcase */
+    const prompts = loadJSON(promptsFile, context.mappings);
+
+    return {
+      prompts: prompts
+    };
+    /* eslint-enable camelcase */
+  }
+
+  return {};
+}
+
+async function dump(context) {
+  const { prompts } = context.assets;
+
+  if (!prompts) return; // Skip, nothing to dump
+
+  const promptsFile = path.join(context.filePath, 'prompts.json');
```

Comment:
The `dump` operation doesn't work for me as it doesn't seem to create the directory for prompts.
Please see `https://github.com/auth0/auth0-deploy-cli/blob/d33413a789864af1ae345ef5debac0e24111d3ff/src/context/directory/handlers/clients.js#L42` for an example.